tests/mocks/requestsMocks.ts

```diff
@@ -0,0 +1,178 @@
+/* eslint-disable  @typescript-eslint/no-magic-numbers */
+import { PolygonPart } from '@map-colonies/mc-model-types';
+import { Polygon } from 'geojson';
+import { PolygonPartsPayload } from '../../src/polygonParts/models/interfaces';
+
+type LayerMetadata = Pick<PolygonPartsPayload, 'catalogId' | 'productId' | 'productType' | 'productVersion'>;
+type PartDataDetails = Omit<PolygonPart, 'footprint'>;
+
+const createLayerMetadata: LayerMetadata = {
+  productId: 'BLUE_MARBLE',
+  productType: 'Orthophoto',
+  catalogId: 'c52d8189-7e07-456a-8c6b-53859523c3e9',
+  productVersion: '1.0',
+};
+
+const updateLayerMetadata: LayerMetadata = {
```

Comment:
you can use spread operator in order only to override the version of `createLayerMetadata`